visualization/app/codeCharta/ui/codeMap/threeViewer/threeCameraService.ts

```diff
@@ -1,49 +1,21 @@
 "use strict"
 
 import { PerspectiveCamera, Vector3 } from "three"
-import { IRootScopeService } from "angular"
-import throttle from "lodash.throttle"
-import { CameraChangeSubscriber, ThreeOrbitControlsService } from "./threeOrbitControlsService"
-import { StoreService } from "../../../state/store.service"
-import { setCamera } from "../../../state/store/appSettings/camera/camera.actions"
-import { CameraService, CameraSubscriber } from "../../../state/store/appSettings/camera/camera.service"
 
-export class ThreeCameraService implements CameraChangeSubscriber, CameraSubscriber {
+export class ThreeCameraService {
 	static VIEW_ANGLE = 45
 	static NEAR = 50
 	static FAR = 200_000 //TODO optimize renderer for far objects
-	private static DEBOUNCE_TIME = 400
-	private readonly throttledCameraChange: () => void
-
 	camera: PerspectiveCamera
-	threeCameraService: jest.Mock
-
-	constructor(private $rootScope: IRootScopeService, private storeService: StoreService) {
-		"ngInject"
-		this.throttledCameraChange = throttle(() => {
-			this.storeService.dispatch(setCamera(this.camera.position), { silent: true })
-		}, ThreeCameraService.DEBOUNCE_TIME)
-		CameraService.subscribe(this.$rootScope, this)
-	}
-
-	onStoreCameraChanged(camera: Vector3) {
-		this.camera.position.set(camera.x, camera.y, camera.z)
-		this.camera.lookAt(0, 0, 0)
-	}
-
-	onCameraChanged() {
-		this.throttledCameraChange()
-	}
 
 	init(containerWidth: number, containerHeight: number) {
 		const aspect = containerWidth / containerHeight
 		this.camera = new PerspectiveCamera(ThreeCameraService.VIEW_ANGLE, aspect, ThreeCameraService.NEAR, ThreeCameraService.FAR)
-		this.setPosition()
-		ThreeOrbitControlsService.subscribe(this.$rootScope, this)
+		this.setPosition(new Vector3(0, 300, 1000))
```

Comment:
I am just wondering where do you got these values here.

Reply:
It was the default value, which got applied initially and whenever an action with undefined was fired from [here](https://github.com/MaibornWolff/codecharta/blob/main/visualization/app/codeCharta/state/store/appSettings/camera/camera.actions.ts#L22)